
class NetworkParameter:
    CLASSES = ['drill_tip']
    ENCODER = 'resnet18'  # 'resnet34'
    ENCODER_WEIGHTS = 'imagenet'
    ACTIVATION = 'sigmoid'
    DEVICE = 'cuda'


